### PR TITLE
Help: Azure DevOps YAML Pipelines

### PR DESCRIPTION
**Pull request by philsturgeon**
I gave up on the previous content which was based on Classic Pipelines because they are being phased out for YAML Pipelines.

So far this is just deploy, but if you like this lets merge it and I'll follow up with diff and preview.

### CODE REVIEW

File: src/_help/continuous-integration/azure-devops.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+   - **npm install**: Installs the npm packages defined in your `package.json` file.
+   - **CLI command**: The `bump deploy` command is a placeholder for any CLI command you want to run. Replace `mycli` with the actual command you need (e.g., `npm run build` or `az --version`).
+
+### Step 3: Setup Environment Variables
+
+Azure DevOps handles environment variables a little differently to other CI/CD tools, but it's ok when you know how.
+
+Firstly, pop over to **Pipelines > Library** and click on **\+ Variable Groups**. 
+
+![](/images/guides/azure-devops/add-variable-group.png)
```
Comment (ChristopheDujarric) [inline src/_help/continuous-integration/azure-devops.md:83]: you missed adding this one :D

**Review by ChristopheDujarric — CHANGES_REQUESTED** [review]
Just a few small details!

**Comment by ChristopheDujarric** [comment]
Oh @philsturgeon it looks like something is wrong as the doc doesn't show up in the Help index (additionally to my other comments):
image

**Comment by philsturgeon** [comment]
@ChristopheDujarric All fixed!

**Review by ChristopheDujarric — APPROVED** [review]

**Pull request merged**